docs/my-website/docs/completion/token_usage.md

```diff
@@ -60,7 +62,24 @@ messages = [{"user": "role", "content": "Hey, how's it going"}]
 print(token_counter(model="gpt-3.5-turbo", messages=messages))
 ```
 
-### 4. `cost_per_token`
+### 4. `create_pretrained_tokenizer` and `create_tokenizer`
+
+```python
```

Comment:
this is great! 

Can we add this as a test here - https://github.com/BerriAI/litellm/blob/f7d7d3426191009f199d09e1aadd7be72ca94458/litellm/tests/test_token_counter.py#L52

To make sure we don't have future regressions @Priva28